core/model/job/step/gotolocation/gotolocation_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/openmarketplaceengine/openmarketplaceengine/cfg"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
```

Comment:
actually should be context here, my goland always auto-inserts this  golang.org/x/net/context which i forget to check. |
I will address it myself separately.